main.mjs

```diff
@@ -0,0 +1,11 @@
+import Chart from './lib/Visualization.svelte';
+
+function render() {
+    new Chart({
+        target: document.getElementById('__svelte-dw'),
+        props: window.__DW_SVELTE_PROPS__,
+        hydrate: true
+    });
+}
+
+render();
```

Comment:
what's this file used for?

Reply:
Good catch, I've fixed it. That was just supposed to be rename of `main.js`, the entry point for the Visualization.svelte bundle. 

In that case, both `js` and `mjs` would work, as it is bundled anyways. I renamed it for consistency.